src/main/java/net/mcreator/workspace/elements/VariableType.java

```diff
@@ -35,7 +35,8 @@
 	private String name;
 	private String color;
 	private String blocklyVariableType;
-	private boolean returnTypeOnly;
+	private boolean ignoredByCoverage;

```

Comment:
I understand your idea with renaming `returnTypeOnly` to `ignoredByCoverage`, but this new name could also mean it supports everything. Did you have another reason to rename it?

Reply:
Not really. It's mostly that the alternative would've been to add a "isLocal" field that does the same thing